include/NovelRT/Graphics/D3D12/Utilities/Graphics.D3D12.Utilities.h

```diff
@@ -0,0 +1,17 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root
+// for more information.
+
+#ifndef NOVELRT_GRAPHICS_D3D12_UTILITIES_H
+#define NOVELRT_GRAPHICS_D3D12_UTILITIES_H
+
+#ifdef WIN32
```

Comment:
We can move this to NovelRT.h to reduce duplication and to keep stuff easier to rationalise about